### ISSUE TITLE
Stateless component descriptions are ignored if declaration is exported

### ISSUE THREAD

**Issue by davidtheclark** [reactjs/react-docgen#51]
If I do this, my docBlock component description works as expected:

```js
/**
 * Description here.
 */
function myComponent() { .. }
export default myComponent;
```

However, in *this* situation the description comes out blank:

```js
/**
 * Description here.
 */
export default function myComponent() { .. }
```

Am I misunderstanding something, or is this a bug?

**Comment by fkling** [comment]
Thanks for reporting! Fixed and released with v2.5.2.

**Comment by davidtheclark** [comment]
Excellent, thanks!
